json/src/main/java/com/basistech/rosette/dm/jackson/BaseAttributeMixin.java

```diff
@@ -25,6 +27,8 @@
  * {@link com.basistech.rosette.dm.BaseAttribute}.
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
+/* These three are used internally as part of 1.0 / 1.1 compatibility, and should not be serialized. */
+@JsonIgnoreProperties({ "old-entity-type", "oldCoreferenceChainId", "oldFlags" })
```

Comment:
Dashes vs. camel case here... but since we're hiding them now, I guess it doesn't matter.
We should have caught in a previous story.